example/http/client/async-ssl/http_client_async_ssl.cpp

```diff
@@ -195,6 +195,18 @@ class session : public std::enable_shared_from_this<session>
 
         // If we get here then the connection is closed gracefully
     }
+
+private:
+    // private constructor called by public construct after strand has been created on the given
+    // note that the same strand is passed as the executor to the constructors of both
+    // resolver_ and stream_.
+    explicit
+    session(net::strand<net::executor> strnd, ssl::context& ctx)
```

Comment:
Go big or go small, nothing in between. So, `strand`, or `ex` (for executor) but not `strnd`. I would go with `ex` since that's what is used everywhere else.